.gitignore

```diff
@@ -35,3 +35,6 @@ lib/generated_plugin_registrant.dart
 
 # Exceptions to above rules.
 !/packages/flutter_tools/test/data/dart_dependencies_test/**/.packages
+
+# GoogleService-Info
+ios/Runner/GoogleService-Info.plist
```

Comment:
Isso é uma chave privada? Se sim, o que acha de encryptar ela e commitar o encryptado? Ai outros desenvolvedores podem baixar e a gente só compartilha a senha de decrypt. 
Se for necessário, claro... Não conheço muito dessa configuração.

Reply:
Eu vi que no seu primeiro commit esse arquivo foi adicionado, se esse conteúdo é sensível, precisava regerar uma nova chave, para invalidar a antiga, pois como foi commitado já, mesmo que tenha removido, está no histórico agora. Acha que é possível gerar uma nova?

Reply:
Eu não tenho permissão para gerar uma nova, acredito só quem criou o projeto teria. Para mudar, precisar ir em https://console.developers.google.com/apis/credentials? e selecionar o App do iOS e gerar uma nova chave.

É necessário gerar também a de Android e também colocar ele no gitignore, pois na branch que foi realizada a configuração do android, o arquivo google-services.json também possui estas informações.

Reply:
Porém, gostaria de conversar com alguém que entende mais de firebase na segunda para entender um pouco mais sobre este arquivo, li um pouco sobre a necessidade de retirar ele - https://stackoverflow.com/questions/44937175/firebase-should-i-add-googleservice-info-plist-to-gitignore
Conforme esta resposta, esta chave vai ficar de qualquer maneira disponível no IPA e no APK do Projeto, e é importante deixar ele fora do projeto pela nossa intenção de deixar ele open source, e quem for utilizar, que crie o próprio arquivo de configuração.

Reply:
Sim, queria ver isso também, olhando o conteúdo dele, não me parece ter nada sensível, me parece que a autenticação se faz pela assinatura do aplicativo mesmo.

Reply:
Cheguei a mesma conclusão lendo esse link [aqui](https://groups.google.com/forum/#!msg/firebase-talk/bamCgTDajkw/uVEJXjtiBwAJ). Me parece que não tem nada crítico no arquivo, porém para projetos open source, acho que o ideal é não versionar esse cara para que cada projeto use suas configurações de firebase...

Reply:
vamos criar um drive pra compartilhar esses arquivos? pior q pro sdk/android eu commitei o arquivo tbm :/

Reply:
Acho que é uma boa criar um drive pra compartilhar e deixar esses arquivos no .gitignore mesmo.

Reply:
Opa, o Drive fo criado pelo Fábio e coloquei os arquivos lá https://drive.google.com/drive/folders/18qG6iaoj4WL_BfVQH1uFV2VYSKf7Mnds